main.nf

```diff
@@ -1490,7 +1493,49 @@ process SPADES {
 }
 
 /*
- * STEP 6.3.1: Run Blast on SPAdes de novo assembly
+ * STEP 6.3.1: Overlap scaffolds with Minimap2, induce and polish assembly, and call variants with seqwish and vg
+ */
+// TODO nf-core: What would the value of $PREFIX by for a multi-fasta?
+process SPADES_VG {
+    tag "$sample"
+    label 'process_medium'
+    publishDir "${params.outdir}/assembly/spades/variants", mode: params.publish_dir_mode,
+        saveAs: { filename ->
+                      if (filename.endsWith(".txt")) "bcftools_stats/$filename"
+                      else filename
+                }
+
+    when:
+    !params.skip_assembly && 'spades' in assemblers && !params.skip_vg
+
+    input:
+    set val(sample), val(single_end), file(scaffolds) from ch_spades_vg
+    file fasta from ch_fasta_spades_vg.collect()
+
+    output:
+    file "*.{paf,gfa,vg,xg,vcf.gz,vcf.gz.tbi}"

```

Comment:
Thinking ahead, it may be useful later to output `paf` to a channel for visualization in https://github.com/tpoorten/dotPlotly, and to output `gfa` to a channel for visualization in https://github.com/rrwick/Bandage and/or [Cytoscape Assembly app](http://apps.cytoscape.org/apps/assembly) or other downstream analysis

Reply:
Using this syntax, all of these files will be written to the results directory anyway so I think we should add explicit channel names only if and when we add these downstream tools to the pipeline.

Reply:
Agreed.  Should perhaps these visualization tools be added to MultiQC instead, as part of common QC for all GFA files?  I could also write a quick GFA summary stats tool if such does not already exist.